rocketmq-docker/4.3.0/docker-compose/docker-compose.yml

```diff
@@ -1,29 +1,23 @@
 version: '2'
 services:
   namesrv:
-    image: apache/rocketmq-namesrv:4.3.0-compose
+    image: huanwei/rocketmq:4.3.0
```

Comment:
Because there is no apache mirror repository (apache/rocketmq), so I decided to use a temporary public mirror repository (huanwei/rocketmq) first, and once the version is stable then they will be migrated to the apache mirror repository.